### PR TITLE
Spelling comments d..e

### PR DESCRIPTION
**Pull request by jsoref**
This PR should be purely comment changes for words (or families of words) that start w/ `d`..`e`

To make my life easier, these initial comment focused PRs will have 2 commits:
* one which is the fixes
* and one which is the copyright bumping

For more information, see #5722 

For reference, these [commits](https://github.com/jsoref/openj9/commits/spelling-full) are generated using https://github.com/jsoref/spelling `f` / `fchurn`. That branch will grow as I make progress (I've finished `s`).

### CODE REVIEW

File: runtime/compiler/optimizer/DataAccessAccelerator.cpp [hunk 1]
```diff
@@ -1150,7 +1150,7 @@ bool TR_DataAccessAccelerator::generateI2PD(TR::TreeTop* treeTop, TR::Node* call
 
       if (isByteBuffer)
          {
-         /* We will be creating a precision diamond for the fast / slow path and eliminating the original call.
+         /* We will be creating a precision diamon.d for the fast / slow path and eliminating the original call.
```
Comment (pshipton) [inline runtime/compiler/optimizer/DataAccessAccelerator.cpp]: why the change from diamond to diamon.d?

File: runtime/compiler/optimizer/DataAccessAccelerator.hpp [hunk 2]
```diff
@@ -187,7 +187,7 @@ class TR_DataAccessAccelerator : public TR::Optimization
     *  \detail
     *     Unlike constant precision DAA call inlining which can be done in-place without introducing extra blocks,
     *     each variable precision call node has to be bloated into multiple
-    *     basic blocks to form a precision diamon. This disrupts the CFG and invalidates block iterator and
+    *     basic blocks to form a precision diamon.. This disrupts the CFG and invalidates block iterator and
```
Comment (pshipton) [inline runtime/compiler/optimizer/DataAccessAccelerator.hpp]: Why the extra `.`?

File: runtime/compiler/env/VMJ9.cpp [hunk 3]
```diff
@@ -4492,7 +4492,7 @@ TR_J9VMBase::initializeLocalArrayHeader(TR::Compilation * comp, TR::Node * alloc
    else if (!comp->getOptions()->realTimeGC() && instanceSize == 0)
       {
       // Contiguous size field is zero (mandatory)
-      // For J9VM_GC_COMBINATION_SPEC only 0 size discontinous arrays are supported
+      // For J9VM_GC_COMBINATION_SPEC only 0 size discontinuous arrays are supported
```
Comment (0xdaryl) [inline runtime/compiler/env/VMJ9.cpp]: The correct term should be "discontiguous".  It was wrong in the first place.

File: runtime/compiler/x/codegen/X86PrivateLinkage.cpp [hunk 4]
```diff
@@ -1667,7 +1667,7 @@ TR::Register *TR::X86PrivateLinkage::buildIndirectDispatch(TR::Node *callNode)
          */
          TR::MemoryReference  *sourceMR = generateX86MemoryReference(vftChild, cg());
          TR::Register *reg = cg()->allocateRegister();
-         // as vftChild->getOpCode().isLoadIndirect is true here, need set excpetionpoint
+         // as vftChild->getOpCode().isLoadIndirect is true here, need set exceptionpoint
```
Comment (pshipton) [inline runtime/compiler/x/codegen/X86PrivateLinkage.cpp]: Should this be exceptionPoint or ExceptionPoint? @fjeremic

File: runtime/gc_verbose_old_events/VerboseEventMetronomeCycleStart.cpp [hunk 5]
```diff
@@ -74,7 +74,7 @@ MM_VerboseEventMetronomeCycleStart::formattedOutput(MM_VerboseOutputAgent *agent
 	char timestamp[32];
 	/* Intervalms for cycle start is the distance (in time) from previous cycle end and this cycle start.
 	 * There should be no heartbeat nither syncGC/OOM events in between the two, since no GC should be happening in between.
-	 * SyncGC due to Explict GC are possible between cycle end and cycle start */
+	 * SyncGC due to Explicit GC are possible between cycle end and cycle start */
```
Comment (pshipton) [inline runtime/gc_verbose_old_events/VerboseEventMetronomeCycleStart.cpp]: This should be `explicit`

File: runtime/gc_verbose_old_events/VerboseEventMetronomeTriggerStart.cpp [hunk 6]
```diff
@@ -74,7 +74,7 @@ MM_VerboseEventMetronomeTriggerStart::formattedOutput(MM_VerboseOutputAgent *age
 	char timestamp[32];
 	/* Intervalms for trigger start is the distance (in time) from previous trigger end and this trigger start.
 	 * There should be no heartbeat neither syncGC/OOM events in between the two, since no GC should be happening in between.
-	 * SyncGC due to Explict GC are possible between trigger end and trigger start */
+	 * SyncGC due to Explicit GC are possible between trigger end and trigger start */
```
Comment (pshipton) [inline runtime/gc_verbose_old_events/VerboseEventMetronomeTriggerStart.cpp]: `explicit`

File: runtime/compiler/x/codegen/J9TreeEvaluator.cpp [hunk 7]
```diff
@@ -6127,7 +6127,7 @@ static void genHeapAlloc(
             //       further instructions.
             //
             // Actually, we should never get here because we've already checked
-            // constant lengths for discontiguity...
+            // constant lengths for discontinuity...
```
Comment (0xdaryl) [inline runtime/compiler/x/codegen/J9TreeEvaluator.cpp]: The original term is correct in the context of arrays.

File: runtime/compiler/optimizer/DataAccessAccelerator.cpp (hunk 1 above)
Comment (jsoref) [reply runtime/compiler/optimizer/DataAccessAccelerator.cpp]: That's damage... My notes says I was trying to replace `diamon.` with `diamond.` and apparently failed miserably: https://github.com/eclipse/openj9/commit/04f3beec2f0d7bc0d29a58232a6bbf8899a9c276#diff-c5d96ecb7f8a1e47f49bf4df0c5f880dR190

File: runtime/compiler/z/codegen/J9TreeEvaluator.cpp [hunk 8]
```diff
@@ -8877,7 +8877,7 @@ J9::Z::TreeEvaluator::VMnewEvaluator(TR::Node * node, TR::CodeGenerator * cg)
                   debugObj->addInstructionComment(cursor, "Denotes start of OOL for allocating zero size arrays");
 
                   /* using TR::Compiler->om.discontiguousArrayHeaderSizeInBytes() - TR::Compiler->om.contiguousArrayHeaderSizeInBytes()
-                   * for byte size for discontinous 0 size arrays because later instructions do ( + 15 & -8) to round it to object size header and adding a j9 class header
+                   * for byte size for discontinuous 0 size arrays because later instructions do ( + 15 & -8) to round it to object size header and adding a j9 class header
```
Comment (0xdaryl) [inline runtime/compiler/z/codegen/J9TreeEvaluator.cpp]: The correct term should be "discontiguous". It was wrong in the first place.

File: runtime/gc_glue_java/ArrayletObjectModel.hpp [hunk 9]
```diff
@@ -127,7 +127,7 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 	}
 
 	/**
-	 * Get the number of discontinous arraylets for the given indexable object
+	 * Get the number of discontinuous arraylets for the given indexable object
```
Comment (0xdaryl) [inline runtime/gc_glue_java/ArrayletObjectModel.hpp]: The correct term should be "discontiguous". It was wrong in the first place.

File: runtime/compiler/optimizer/DataAccessAccelerator.hpp (hunk 2 above)
Comment (jsoref) [reply runtime/compiler/optimizer/DataAccessAccelerator.hpp]: I'll update all of these once you get an answer to your question below. (They're locally fixed)

File: runtime/zip/zipsup.c [hunk 10]
```diff
@@ -143,7 +143,7 @@ I_32 initZipLibrary(J9PortLibrary* portLib, char* dir)
 		j9str_printf(portLib, correctPathPtr, expectedPathLength, "%s/%s", dir, J9_ZIP_DLL_NAME);
 		if(j9sl_open_shared_library(correctPathPtr, &zipDLLDescriptor, TRUE)) goto openFailed;
 	} else {
-		/* dir is NULL. It shouldn't happen, but in case, revert back to original dlload that
+		/* dir is NULL. It shouldn't happen, but in case, revert back to original dllload that
```
Comment (pshipton) [inline runtime/zip/zipsup.c]: I think this should be dlopen or dlopen()

File: sourcetools/j9nls/com/ibm/oti/NLSTool/J9NLS.java [hunk 11]
```diff
@@ -776,7 +776,7 @@ public void appendMsg(String moduleName, String key) {
 		public void writeHeader(String headerName, String moduleName) {
 			String header = headerName.replace('.', '_');
 			append("/*\n");
-			append(" * Copyright (c) 1991, 2017 IBM Corp. and others\n");
+			append(" * Copyright (c) 1991, 2019 IBM Corp. and others\n");
```
Comment (pshipton) [inline sourcetools/j9nls/com/ibm/oti/NLSTool/J9NLS.java]: Please revert this to 2017

Comment (jsoref) [reply sourcetools/j9nls/com/ibm/oti/NLSTool/J9NLS.java]: @fjeremic -- fwiw, I'm pretty sure your script would also trip on this...

Comment (fjeremic) [reply sourcetools/j9nls/com/ibm/oti/NLSTool/J9NLS.java]: Yikes! Good catch!

File: runtime/compiler/x/codegen/X86PrivateLinkage.cpp (hunk 4 above)
Comment (fjeremic) [reply runtime/compiler/x/codegen/X86PrivateLinkage.cpp]: Should be "exception point".

File: runtime/compiler/env/VMJ9.cpp (hunk 3 above)
Comment (jsoref) [reply runtime/compiler/env/VMJ9.cpp]: fwiw, those are from: e8cc1e8, ae808d4, and 9c69e1b.
Thanks for the correction.
